core/src/main/java/io/grpc/transport/HttpUtil.java

```diff
@@ -189,15 +189,22 @@ public static Http2Error forCode(long code) {
     }
 
     /**
-     * Looks up the {@link Status} from the given HTTP/2 error code.
+     * Looks up the {@link Status} from the given HTTP/2 error code. This is preferred over {@code
+     * forCode(code).status()}, to more easily conform to HTTP/2:
+     *
+     * <blockquote><code>Unknown or unsupported error codes MUST NOT trigger any special behavior.
```

Comment:
I guess we don't need code, since it is just text (<code> does make it look like an RFC though :-) ).